### ISSUE TITLE
None object is not subscriptable

### ISSUE THREAD

**Issue by iceb0y** [vijos/vj4#151]
e.g. https://vijos.org/discuss/5109fdea4e41125014000bb5

```
Apr  5 14:35:38 vj4 python3.5[385]: #033[31m[E 170405 14:35:38 web_protocol:358]#033[0m Error handling request
Apr  5 14:35:38 vj4 python3.5[385]: Traceback (most recent call last):
Apr  5 14:35:38 vj4 python3.5[385]:   File "/usr/local/lib/python3.5/dist-packages/aiohttp/web_protocol.py", line 417, in start
Apr  5 14:35:38 vj4 python3.5[385]:     resp = yield from self._request_handler(request)
Apr  5 14:35:38 vj4 python3.5[385]:   File "/usr/local/lib/python3.5/dist-packages/aiohttp/web.py", line 289, in _handle
Apr  5 14:35:38 vj4 python3.5[385]:     resp = yield from handler(request)
Apr  5 14:35:38 vj4 python3.5[385]:   File "/home/vijos/vj4/vj4/handler/base.py", line 217, in __iter__
Apr  5 14:35:38 vj4 python3.5[385]:     yield from super(Handler, self).__iter__()
Apr  5 14:35:38 vj4 python3.5[385]:   File "/usr/local/lib/python3.5/dist-packages/aiohttp/web_urldispatcher.py", line 657, in __iter__
Apr  5 14:35:38 vj4 python3.5[385]:     resp = yield from method()
Apr  5 14:35:38 vj4 python3.5[385]:   File "/home/vijos/vj4/vj4/handler/discussion.py", line 156, in get
Apr  5 14:35:38 vj4 python3.5[385]:     (vnode['title'], node_url(self, 'discussion_node', discussion.node_id(ddoc))),
Apr  5 14:35:38 vj4 python3.5[385]: TypeError: 'NoneType' object is not subscriptable#033[0m
```

**Comment by iceb0y** [comment]
The problem (1701) does not exist, but the discussion points to it.

**Comment by twd2** [comment]
Fixed by f6afaedd.
